docs/bugs_and_glitches.md

```diff
@@ -391,6 +392,104 @@ Then edit four routines in [engine/battle/effect_commands.asm](https://github.co
 ```
 
 
+## Saves can be corrupted in such a way as to glitch the game
```

Comment:
How about "Turning off the power during a save can corrupt the game"?

Reply:
Well, that's not a problem that can be corrected, since the save game write is not and cannot be atomic. How about "Saves corrupted [by power-off mid-saving] are not handled"?

Reply:
Resolved.